tests/DigestTest.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I believe current guidance is that we no longer need to specify a year. If you prefer to add the year, however, it should be 2020 for a new file.

Reply:
removed year, as it is no longer necessary